madmom/audio/signal.py

```diff
@@ -510,6 +516,9 @@ class Signal(np.ndarray):
     num_channels : int, optional
         Reduce or expand the signal to `num_channels` channels, or 'None'
         to return the signal with its original channels.
+    channel : int, optional
+        When reducing a signal to `num_channels` of 1, use this channel,
+        or 'None' to return the average across all channels.
```

Comment:
Yes I liked this one. IIRC I used a this different description because a lot of them just pass through, so don't _really_ do the logic themselves, but I can propagate this through everywhere